pyalgotrading/algobulls/connection.py

```diff
@@ -709,11 +764,116 @@ def get_papertrading_report_order_history(self, strategy_code):
         assert isinstance(strategy_code, str), f'Argument "strategy_code" should be a string'
         return self.get_report(strategy_code=strategy_code, trading_type=TradingType.PAPERTRADING, report_type=TradingReportType.ORDER_HISTORY)
 
-    def realtrade(self, *args, **kwargs):
+    def realtrade(self, strategy=None, start=None, end=None, instruments=None, lots=None, parameters=None, candle=None, mode=StrategyMode.INTRADAY, delete_previous_trades=True, initial_funds_virtual=1e9, **kwargs):
         """
         Start a Real Trading session.
         Update: This requires an approval process which is currently on request basis.
+
+        Start a real trade job for a strategy on the AlgoBulls Platform
+
+        Args:
+            strategy: Strategy code
+            start: Start date/time
+            end: End date/time
+            instruments: Instrument key
+            lots: Number of lots of the passed instrument to trade on
+            parameters: Parameters
+            candle: Candle interval
+            mode: Intraday or delivery
+            delete_previous_trades: Delete data of all previous trades
+            initial_funds_virtual: virtual funds allotted before the real trading starts
+
+
+        Legacy args (will be deprecated in future release):
+            'strategy_code' behaves same 'strategy'
+            'start_timestamp' behaves same 'start'
+            'end_timestamp' behaves same 'end'
+            'instrument' behaves same 'instruments'
+            'strategy_parameters' behaves same 'parameters'
+            'candle_interval' behaves same 'candle'
+            'strategy__mode' behaves same 'mode'
+
+        Returns:
+            realtrade job submission status
         """
+
+        # check if values received by new parameter names, else extract from old parameter names
```

Comment:
i see code duplication in realtrade; it should be similar to papertrade one; pls implement DRY

Reply:
done